src/styles/sponsors.css

```diff
@@ -2,9 +2,62 @@
     display: flex;
     flex-direction: column;
     text-align: center;
-    background-color: red;
+    gap: 0px;
+    height: 150px;
 }
 
 .thanks{
     font-weight: normal;
+    font-size: x-large;
+}
+
+.sponsor-tiers{
+    display: flex;
+    flex-direction: column;
+    column-gap: normal;
+    justify-content: center;
+    align-items: center; 
+    gap: 50px;
+    margin-left: 50px;
+    margin-right: 50px;
+    max-width: 100%;    
+}
+ .gold-sponsors {
```

Comment:
is it possible to collapse the similar properties of .gold-sponors, .silver-sponsors, and .bronze-sponsors into a single .sponsors? you can keep the specific tags, but combine general properties under one tag instead of having duplicate properties.